TLM/TLM/Util/RoundaboutMassEdit.cs

```diff
@@ -223,15 +253,15 @@ public class RoundaboutMassEdit {
             var segments = GetSortedSegments( segmentId);
 
             foreach (var nextSegmentId in segments) {
-                bool isRAbout = false;
+                bool isRoundabout = false;
                 if (nextSegmentId == segmentList[0]) {
-                    isRAbout = true;
+                    isRoundabout = true;
                 } else if (Contains(nextSegmentId)) {
-                    isRAbout = false;
+                    isRoundabout = false;
```

Comment:
It looks like a bug. `isRoundabout` cannot be `true` here so there is no point to set it to false again

Reply:
@kianzarrin I think you didn't understand my point here. 
I meant that check if nextSegmentId is inside list is completely pointless because you don't do anything regardless of result. That's why is think it is a bug here, because why bother checking for specific case when no different action is performed if occurs?